components/Wrapper/index.js

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+
+const StyleWrapper = styled.div`
+  margin-bottom: 30px;
+`;
+
+export default function Wrapper({ children }) {
+  return <StyleWrapper>{children}</StyleWrapper>;
+}
```

Comment:
you can export the StyledWrapper instead of creating this component (its absolutely the same ;) )